server/sv_main.lua

```diff
@@ -108,9 +108,7 @@ end)
 
 RegisterCommand('togglealerts', function(source, args, user)
 	local source = source
-    local Player = QBCore.Functions.GetPlayer(source)
-	local job = Player.PlayerData.job
-	if IsPoliceJob(job.name) or job.name == 'ambulance' then
+	if IsDispatchJob then
```

Comment:
This is removing Police jobs and replacing for Dispatch job?

Reply:
that is checking for if someone is LEO or EMS, i figured it would be better to use an already made function then write one and the IsPoliceJob function only checks police so if someone was EMS they wouldn't have access to toggle the alerts.

Reply:
> that is checking for if someone is LEO or EMS, i figured it would be better to use an already made function then write one and the IsPoliceJob function only checks police so if someone was EMS they wouldn't have access to toggle the alerts.

I don't see where IsDispatchJob is being called, can you link me? I recon we could switch it to 

Config.PoliceJobs or Config.AmbulanceJobs

Reply:
One last time, github hopefully will chill now and not freak out again, this is the IsDispatchJob function https://github.com/Project-Sloth/ps-dispatch/blob/5bb0f9e44fdf951a19327b69e64c3b48553459b5/server/sv_main.lua#L18-L32

Reply:
> One last time, github hopefully will chill now and not freak out again, this is the IsDispatchJob function
> 
> https://github.com/Project-Sloth/ps-dispatch/blob/5bb0f9e44fdf951a19327b69e64c3b48553459b5/server/sv_main.lua#L18-L32

I was looking for this on ps-mdt, all this time I thought I was working on ps-mdt... sorry. Lol

This is all good